publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RabbitMqProperties.java

```diff
@@ -416,15 +427,17 @@ private boolean hasExchange() throws RemRemPublishException {
         try {
             channel = connection.createChannel();
         } catch (final IOException e) {
-            log.info("Exchange " + exchangeName + " does not Exist");
+            log.info("Exchange " + exchangeName + " does not exist");
             throw new RemRemPublishException("Exception occurred while creating Channel with Rabbitmq connection ::" + factory.getHost() + factory.getPort() + e.getMessage());
         }
         try {
             channel.exchangeDeclarePassive(exchangeName);
-            return true;
+            hasExchange = true;
+            return hasExchange;
         } catch (final IOException e) {
-            log.info("Exchange " + exchangeName + " does not Exist");
-            return false;
+            log.info("Exchange " + exchangeName + " does not exist");
+            hasExchange = false;
+            return hasExchange;
```

Comment:
Maybe we can write
  return false;
The result is the same, but it emphasizes that 'false' is returned, i.e. there's no need to find out what is the value of hasExchange.